doc/Makefile

```diff
@@ -151,13 +151,3 @@ features.rst: $(shell find ../pylint/checkers -type f -regex '.*\.py')
 	PYTHONPATH=$(PYTHONPATH) $(PYTHON) ./exts/pylint_features.py
 
 messages: $(PYTHONPATH) $(PYTHON) ./exts/pylint_messages.py
-
-gen-examples:
```

Comment:
We should generate the example as toml now, right ?

Reply:
We don't really use this.

Should probably take another look at this with that PR/issue about config documentation. For now this is unused.

Reply:
Yeah I think we have an outdated example of pylintrc because of this. I think it was supposed to be updated periodically. I would remove this script and add the equivalent inside the tbump.toml

Reply:
I think it can be easily done with an extension for Sphinx like we have for messages. I'll take a stab at this after `argparse` has been done to fix all of our documentation about this!

Reply:
https://github.com/PyCQA/pylint/pull/6282/commits/c284444c3f18dd2c761233a9f2ee6bac88e8b172